scene/gui/code_edit.cpp

```diff
@@ -1526,6 +1526,7 @@ void CodeEdit::_clear_delimiters(DelimiterType p_type) {
 		}
 	}
 	delimiter_cache.clear();
+	_update_delimiter_cache();
```

Comment:
Should only do this if `setting_delimiters == false` else we'll calculate it twice in short succession, which can be quite expensive.